src/core/border_router/routing_manager.hpp

```diff
@@ -602,6 +602,39 @@ class RoutingManager : public InstanceLocator
 
 #endif // OPENTHREAD_CONFIG_BORDER_ROUTING_DHCP6_PD_ENABLE
 
+#if OPENTHREAD_CONFIG_BORDER_ROUTING_REACHABLITY_CHECK_ICMP6_ERROR_ENABLE
+    /**
+     * Determines whether to send an ICMPv6 Destination Unreachable error to the sender based on reachability and
+     * source address.
+     *
+     * If the Border Router (BR) decides to forward an IPv6 message to a destination outside the Adjacent Infrastructure
+     * Link (AIL),  and the message's source address matches a BR-generated ULA OMR prefix (with low preference) an
+     * ICMPv6 Destination Unreachable message is sent to the sender.
+     *
+     * This check is performed only when a local, non-infrastructure-derived ULA OMR prefix is published and the BR
```

Comment:
```suggestion
     * The ICMPv6 error message is only sent when the IPv6 source matches a local, non-infrastructure-derived ULA OMR prefix being published and the BR
```

Reply:
I decided to remove this paragraph now, since this is alrady covered by the previous paragraph (and we no longer require `::/0` to be publsihed by this BR).